opal/include/opal/sys/arm64/atomic.h

```diff
@@ -99,7 +99,7 @@ static inline bool opal_atomic_compare_exchange_strong_acq_32(opal_atomic_int32_
     __asm__ __volatile__("1:  ldaxr   %w0, [%2]       \n"
                          "    cmp     %w0, %w3        \n"
                          "    bne     2f              \n"
-                         "    stxr    %w1, %w4, [%2]  \n"
+                         "    stlxr   %w1, %w4, [%2]  \n"
```

Comment:
Why do we need release semantics for strong_acq_ ?

Reply:
Yes, you are right.  strong_acq no need to change, only  exchange_strong_ need change to stlxr.  I updated